docs/docs/getting-started/first-steps.md

```diff
@@ -13,19 +13,47 @@ If you encounter any problem with the following steps, make sure to use the [lat
 
 ## Create a cluster
 
-1. Create the configuration file and IAM resources for your selected cloud provider
+1. Create the [configuration file](../workflows/config.md)for your cloud provider.
```

Comment:
```suggestion
1. Create the [configuration file](../workflows/config.md) for your cloud provider
```

Reply:
what do you think of this change @thomasten ?

Reply:
You mean Mia's change? That's basically what I suggested.
You mean your suggestion? Space :+1:, removal of period :-1: (see below)

Reply:
i meant shortening the first steps